mod1/mod11/src/file-6.xoo

```diff
@@ -0,0 +1,11 @@
+// dummy line 1
+// dummy line 2
+// dummy line 3
+// dummy line 4
+// dummy line 5
+// dummy line 6
+// dummy line 7
+// dummy line 8
+// dummy line 9
+// dummy line 10
+// violate HasTag rule: xoo on line 11
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:19188/organizations/org4/rules#rule_key=xoo%3AHasTag))

[See it in SonarQube](http://127.0.0.1:19188/project/issues?id=sample&branch=feature5&issues=AWEt9T7MFJlJ4KJ2TUs3&open=AWEt9T7MFJlJ4KJ2TUs3)

[ISSUE_KEY]: AWEt9T7MFJlJ4KJ2TUs3

Reply:
This issue has been resolved by a user in SonarQube

[ISSUE_STATUS]: RESOLVED